tfx/components/statistics_gen/executor_test.py

```diff
@@ -105,6 +115,8 @@ def testDoWithSchemaAndStatsOptions(self):
     exec_properties = {
         executor.STATS_OPTIONS_JSON_KEY:
             tfdv.StatsOptions(label_feature='company').to_json(),
+        executor.EXCLUDE_SPLITS_KEY:
+            None
```

Comment:
it won't be None for executor (as component will set it to default), same elsewhere

Reply:
yes, to be consistent with schema gen and example validator, i think we also remove the case for exclude_splits = None for statsgen